addons/mail/models/res_config_settings.py

```diff
@@ -42,6 +43,12 @@ class ResConfigSettings(models.TransientModel):
     )
     sfu_server_url = fields.Char("SFU Server URL", config_parameter="mail.sfu_server_url")
     sfu_server_key = fields.Char("SFU Server key", config_parameter="mail.sfu_server_key", help="Base64 encoded key")
+    sfu_local_key = fields.Char(
+        "SFU Local key",
+        config_parameter="mail.sfu_local_key",
+        default=lambda self: str(uuid.uuid4()),
+        help="Key for encryption with the SFU"
+    )
```

Comment:
Isn't this effectively useless if not displayed in the view?
ir config param are not automatically created from this.

Reply:
And even with the view, wouldn't we want this value to be automatically set on database create/upgrade instead?